datadog_checks_dev/datadog_checks/dev/plugin/pytest.py

```diff
@@ -181,12 +180,12 @@ def run_check(config=None, **kwargs):
 
         result = run_command(check_command, capture=True)
 
-        matches = re.findall(AGENT_COLLECTOR_SEPARATOR + r'\n(.*?\n(?:\} \]|\]))', result.stdout, re.DOTALL)
+        matches = re.findall(r'\n(.*?\n(?:\} \]|\]))', result.stdout, re.DOTALL)
```

Comment:
I think that the `=== JSON ===` anchor was used by the regex as a marker of the beginning of the JSON document in order to skip the logs that are preceding it.

We need to replace it otherwise, the regex starting by `\n.*…` will collect the logs.

```suggestion
        matches = re.findall(r'\n((?:\{ \[|\[).*?\n(?:\} \]|\]))', result.stdout, re.DOTALL)
```